rain_api_core/view_util.py

```diff
@@ -163,6 +164,12 @@ def decode_jwt_payload(jwt_payload, algo=JWT_ALGO):
     except jwt.InvalidSignatureError as e:
         log.info('JWT has failed verification. returning empty dict')
         return {}
+
+    set_jwt_blacklist()
```

Comment:
For aesthetic reason, can we move this into `is_jwt_blacklisted()`?

Reply:
Sure 👍